### PR TITLE
Testem update

### PR DESCRIPTION
**Pull request by stopdropandrew**
This should fix the testem-electron.js issue.

### CODE REVIEW

File: lib/utils/test-task-mixin.js [hunk 1]
```diff
@@ -1,25 +1,9 @@
...
-  },
-
-  defaultOptions() {
-    return setTestemFile(this._super(...arguments));
-  },
-
-  testemOptions() {
-    return setTestemFile(this._super(...arguments)) || this.defaultOptions(...arguments);
+  run(options) {
+    options.configFile = 'testem-electron.js';
```
Comment (jacobq) [inline lib/utils/test-task-mixin.js:5]: Shouldn't we still allow the consumer to override if they want? e.g. `options.configFile = options.configFile || 'testem-electron.js';`

**Comment by jacobq** [comment]
Thanks for the PR to my PR ;)
I will try it out.

**Comment by jacobq** [comment]
FYI it [looks like](https://travis-ci.org/felixrieseberg/ember-electron/jobs/402562038#L1108) this doesn't fix the [problem importing `ember-cli/lib/models/blueprint`](https://github.com/felixrieseberg/ember-electron/pull/354#issuecomment-403650859).

File: lib/utils/test-task-mixin.js (hunk 1 above)
Comment (stopdropandrew) [reply lib/utils/test-task-mixin.js:5]: Good call!

**Pull request merged**